src/foam/nanos/auth/resetPassword/ResetPasswordTokenService.js

```diff
@@ -107,10 +107,13 @@ return true;`
 // context. Therefore we put the system user in the context here so that
 // decorators down the line won't throw NPEs when trying to access the user in
 // the context.
+AppConfig appConfig = (AppConfig) x.get("appConfig");
 User systemUser = (User) getX().get("user");
 x = x.put("user", systemUser);
 
+
 String newPassword = user.getDesiredPassword();
+String url = appConfig.getUrl().replaceAll("/$", "");
```

Comment:
can you make this group.appConfig pls.
like: https://github.com/nanoPayinc/NANOPAY/pull/9243/files

Reply:
re: appconfig - Session.applyTo has been updated to do this for us, so pulling appConfig from the context is fine.  Thanks for catching that Anna.

Reply:
actually we should just do this when setting up AppConfig.  I'll do that.

Reply:
not as straightforward as I thought, we'll leave this for now.